core/templates/dev/head/services/AssetsBackendApiService.js

```diff
@@ -67,7 +67,11 @@ oppia.factory('AssetsBackendApiService', [
         timeout: canceler.promise
       }).success(function(data) {
         try {
-          var assetBlob = new Blob([data]);
+          if (assetType === ASSET_TYPE_AUDIO) {
```

Comment:
Can we add a comment here that why we have hardcoded the type of audio blob?

Reply:
Done, sounds good.